spec/grape-swagger/entities/response_model_spec.rb

```diff
@@ -96,6 +96,23 @@ class Tag < Grape::Entity
           expose :name, documentation: { type: 'string', desc: 'Name' }
         end
 
+        class Nested < Grape::Entity
+          expose :nested, documentation: { type: 'Object', desc: 'Nested entity' } do
```

Comment:
please change one `object` to `Hash`, so others can see, this would be used synonym